types/src/transaction/transaction.rs

```diff
@@ -225,6 +316,15 @@ impl Transaction {
                 Some(amount) if amount == 0 => Err(Error::ZeroAmount),
                 _ => Ok(()),
             },
+            Transaction::AssetCompose {
+                inputs,
+                ..
+            } => {
+                if inputs.len() == 0 {
+                    return Err(Error::EmptyInput)
+                }
+                Ok(())
```

Comment:
Let's check the amounts of both input and output aren't zero.
This constraint was added after my work.

Reply:
The type of output is `output: AssetMintOutput,`.What kind of length are you talking about?

Reply:
I meant `amount` of the inputs and the output. Please refer to the verification of `AssetMint` and `AssetTransfer`

Reply:
Okay, I added check routine.